src/m365/spo/commands/folder/folder-roleassignment-add.ts

```diff
@@ -180,29 +180,14 @@ class SpoFolderRoleAssignmentAddCommand extends SpoCommand {
         const siteUser = await spo.ensureEntraGroup(args.options.webUrl, group);
         principalId = siteUser.Id;
       }
-
-      await this.breakRoleAssignment(requestUrl);
+      
       await this.addRoleAssignment(requestUrl, principalId!, roleDefinitionId);
     }
     catch (err: any) {
       this.handleRejectedODataJsonPromise(err);
     }
   }
 
-  private async breakRoleAssignment(requestUrl: string): Promise<void> {
```

Comment:
There are some references to the `breakroleinheritance` api in the test files as well. Could you remove those?